android/app/src/androidTest/java/com/trustwallet/core/app/blockchains/terra/TestTerraTransactions.kt

```diff
@@ -99,111 +98,17 @@ class TestTerraTransactions {
         val signingInput = Cosmos.SigningInput.newBuilder().apply {
             signingMode = SigningMode.Protobuf
             accountNumber = 3407705
-            chainId = "columbus-5"
+            chainId = "phoenix-1"
             memo = ""
             sequence = 3
             fee = cosmosFee
             privateKey = ByteString.copyFrom(key.data())
             addAllMessages(listOf(message))
         }.build()
 
-        val output = AnySigner.sign(signingInput, TERRA, SigningOutput.parser())
-
-        assertEquals(output.serialized, "{\"mode\":\"BROADCAST_MODE_BLOCK\",\"tx_bytes\":\"CucBCuQBCiYvdGVycmEud2FzbS52MWJldGExLk1zZ0V4ZWN1dGVDb250cmFjdBK5AQosdGVycmExOHd1a3A4NGRxMjI3d3U0bWdoMGptNm45bmxuajZyczgycHA5d2YSLHRlcnJhMTR6NTZsMGZwMmxzZjg2enkzaHR5Mno0N2V6a2hudGh0cjl5cTc2Glt7InRyYW5zZmVyIjp7ImFtb3VudCI6IjI1MDAwMCIsInJlY2lwaWVudCI6InRlcnJhMWpsZ2FxeTludm4yaGY1dDJzcmE5eWN6OHM3N3duZjlsMGttZ2NwIn19EmcKUApGCh8vY29zbW9zLmNyeXB0by5zZWNwMjU2azEuUHViS2V5EiMKIQNwZjrHsPmJKW/rXOWfukpQ1+lOHOJW3/IlFFnKLNmsABIECgIIARgDEhMKDQoFdWx1bmESBDMwMDAQwJoMGkAaprIEMLPH2HmFdwFGoaipb2GIyhXt6ombz+WMnG2mORBI6gFt0M+IymYgzZz6w1SW52R922yafDnn7yXfutRw\"}")
-        assertEquals(output.error, "")
-    }
-
-    @Test
-    fun testSigningWasmTerraGenericProtobuf() {
```

Comment:
move to luna classic test?

Reply:
I have removed the Terra classic specific tests (swift, kotlin), not relevant any more